sdk/go/common/resource/urn/urn.go

```diff
@@ -201,5 +201,26 @@ func (urn URN) Rename(newName string) URN {
 
 // Returns a new URN with an updated stack part
 func (urn URN) RenameStack(stack tokens.StackName) URN {
-	return New(stack.Q(), urn.Project(), urn.QualifiedType(), urn.Type(), urn.Name())
+	return New(
+		stack.Q(),
+		urn.Project(),
+		// parent type is empty because
+		// assuming the qualified type already includes it
```

Comment:
```suggestion
		// the qualified type already includes it
```

This probably is fine on one line

Reply:
Yeah, fixed the place I copy pasted from as well :sweat_smile: